### ISSUE TITLE
mat-alert is read by the NVDA reader twice 

### ISSUE THREAD

**Issue by AliciaThomson** [angular/components#19512]
Input placeholder is read by the NVDA reader twice

#### Reproduction

Use StackBlitz to reproduce your issue: https://stackblitz.com/edit/angular-q3ssql

Steps to reproduce:
1. Go to https://stackblitz.com/edit/angular-q3ssql
2. Run NVDA reader (default settings)
3. Focus on first field 
4. Then tab to next field too trigger error on first field
 
#### Expected Behavior

mat-error message should only read once.

After step 3: focus on first field
screen reader reads:
```
[ mat-label from first field ]  edit  required  has auto complete  [ placeholder from first field ] 
```

After step 4: tab to next field too trigger error on first field
screen reader reads:
```
[ mat-error from first field] 
invalid entry
[ mat-label from second field ]  edit  required  has auto complete [ placeholder from second field ]
```

#### Actual Behavior

What behavior did you actually see?

After step 3: focus on first field
screen reader reads:
```
[ mat-label from first field ]  edit  required  has auto complete  [ placeholder from first field ] 
```

After step 4: tab to next field too trigger error on first field
screen reader reads:
```
[ mat-error from first field] 
invalid entry
[ mat-label from second field ]  edit  required  has auto complete [ placeholder from second field ]
alert [ mat-error from first field ]
```

Screen reader experience from example:
```
Enter your name  edit  required  has auto complete  pat@example.com
You must enter a value
invalid entry
Enter your email  edit  required  has auto complete  pat@example.com
alert  You must enter a value
```

**Comment by jelbourn** [comment]
Duplicate of #14281
